core/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -80,16 +80,18 @@ public AdminUtil(boolean exitOnError) {
     private final List<String> hlocks;
     private final List<String> wlocks;
     private final String top;
+    private final String timeCreated;
 
     private TransactionStatus(Long tid, TStatus status, String debug, List<String> hlocks,
-        List<String> wlocks, String top) {
+        List<String> wlocks, String top, String timeCreated) {
 
       this.txid = tid;
       this.status = status;
       this.debug = debug;
       this.hlocks = Collections.unmodifiableList(hlocks);
       this.wlocks = Collections.unmodifiableList(wlocks);
       this.top = top;
+      this.timeCreated = timeCreated;
```

Comment:
Did you give any thought to storing / passing the timeCreated as a long?  That could make use of the transaction status easier for anything that wanted to access the timestamp for other purposes.  This would defer the formatting closer to where its needed and other uses would not need to parse the string.  Sorry that this didn't occur to me earlier.

Reply:
What should be in the catch block inside `ZooStore` then? Should I just return 0 or 1/-1 or revert back to throwing the runtimeException?

Reply:
I guess 0?  The if > 0 the print can format the time otherwise ERROR